src/coreclr/src/inc/bundle.h

```diff
@@ -34,7 +34,7 @@ struct BundleFileLocation
     bool IsValid() const { LIMITED_METHOD_CONTRACT; return Offset != 0; }
 };
 
-typedef bool(__stdcall BundleProbe)(LPCSTR, INT64*, INT64*);
+typedef bool(BundleProbe)(LPCSTR, INT64*, INT64*);
```

Comment:
Should we move this and the new one to https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/coreclr/src/hosts/inc/coreclrhost.h

It's basically part of the ABI. Maybe just as typedef, no comments/docs.